sdk/core/azure-core/inc/azure/core/exception.hpp

```diff
@@ -145,10 +145,9 @@ namespace Azure { namespace Core {
      *
      */
     ~RequestFailedException() = default;
-
-  private:
-    std::string GetRawResponseField(
-        std::unique_ptr<Azure::Core::Http::RawResponse>& rawResponse,
-        std::string fieldName);
   };
+
+  std::string GetRawResponseField(
```

Comment:
If we really want to make this public, which I am not sure we do atm, I don't think exposing it from outside of the scope of a class like RFE (or probably RawResponse), makes sense.